### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Jimbo4350**

### CODE REVIEW

File: cabal.project [hunk 1]
```diff
@@ -171,43 +171,43 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-ledger-specs
-  tag: 88abb4da0901aaea540dc883d70f8f7a49184551
-  --sha256: 1gyi5j7d1rjq2qs7g4rqwxzrjrgs2p3jbkjbapla5yz3h2bmrhs4
+  tag: b687fa55369f480cb134c007e534662bef961cea
+  --sha256: 1bwbrj22a0nmsw990ac8k4ffz4xdazfb85dyl3lv3pa5yp3svw1j
```
Comment (mrBliss) [inline cabal.project:175]: Note that `cardano-ledger` has been merged in `cardano-ledger-specs`, so all mentions of `cardano-ledger` should be replaced with `cardano-ledger-specs`.

The new subdirs are:
```
byron/ledger/impl
byron/ledger/impl/test
byron/crypto
byron/crypto/test
```

(Don't forget to update the `stack.yaml` too)

**Comment by dcoutts** [comment]
bors merge

**Comment by dcoutts** [comment]
bors merge

**Comment by dcoutts** [comment]
bors merge

**Pull request merged**